### ISSUE TITLE
Weather sporadically turns into the People icon

### ISSUE THREAD

**Issue by PimpUigi** [valinet/ExplorerPatcher#2010]
Seems like everything is fine and we're just chilling, then you go to look at the Weather on the Taskbar and *poof* it's just the People icon.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Comment by PimpUigi** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/7344271/fcb467fc-080e-45ec-84f0-d9bfd05b2828)
![image](https://github.com/valinet/ExplorerPatcher/assets/7344271/0819f957-9268-409c-9215-ce90ac433e21)
![image](https://github.com/valinet/ExplorerPatcher/assets/7344271/0867599d-9a6a-4447-bd49-e3028c991b28)
![image](https://github.com/valinet/ExplorerPatcher/assets/7344271/2f75073e-7841-4679-a46f-9a10657b6ca4)

How many of these Explorer Patcher settings pages do you want? The rest of them?

**Comment by PimpUigi** [comment]
Just realized I can do this:
[ExplorerPatcher_settings22621.1992.56.3.zip](https://github.com/valinet/ExplorerPatcher/files/12550261/ExplorerPatcher_settings22621.1992.56.3.zip)

**Comment by pyrates999** [comment]
That is a preview update: https://support.microsoft.com/en-us/topic/august-22-2023-kb5029351-os-build-22621-2215-preview-9af25662-083a-43f5-b3a7-975fe25cc692

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

Preview updates are not supported by EP.

**Comment by PimpUigi** [comment]
Well, that was actually me trying to troubleshoot the issue on my own. I've been experiencing it for about six months or so?
At some point I was like 'Yeah lets do preview updates cause it might be fixed there.' and whelp, it is what it is. I recently did uninstall and reinstall because of one of those windows updates that required me to do so.

If there's not an issue you can find, hopefully it shakes out next major Windows version. Who knows. I tend to be patient.

**Comment by pyrates999** [comment]
Uninstall EP and see if this is still happening.

**Comment by Jazzzed** [comment]
@pyrates999 Yes I have also seen this occur with Windows build 22621.2134 when running the current version of ExplorerPatcher. It is intermittent. One way I saw it occur was when I swiped my mouse cursor over the top of it while traveling to some other location. That is not a sure fire way to cause it to happen though. As said, it appears to be intermittent. I have also seen this in previous Windows builds going back to at least March.

**Comment by pyrates999** [comment]
So just to confirm, this does not happen if you uninstall EP and reboot?

**Comment by Jazzzed** [comment]
It occurs no matter what. It is a bug. Uninstalling / reinstalling ExplorerPatcher has no effect on killing the bug permanently. It will reappear, but the circumstances as to when or if it will appear is random. Reboots fix the problem for a while, but it will often turn into the people icon at some random point or if I happen to hit it with the mouse cursor at a moment when it doesn't want that.

**Comment by pyrates999** [comment]
Then please close this as it is not a bug in EP.

**Comment by Jazzzed** [comment]
Yes it is. Windows 11 doesn’t even contain a weather widget. This is 100% a bug with the Windows 10 taskbar in ExplorerPatcher. And wow, I was just about to donate to the cause, but it is obvious that no one takes this development seriously. This program can and is currently trainwrecking people’s operating systems. Not this bug in particular, but if you disregard every single person’s reports of issues, then what is the point? I’m out too. This program and the people behind it are dangerous.

**Comment by PimpUigi** [comment]
> Then please close this as it is not a bug in EP.

Yeah so it only took me the day or two to have this happen again after uninstall/reinstall (which is why my reply is this late).
This reply from you confuses me though? Clearly two of us have the bug. xD
Workaround is just restarting File Explorer and then it will happen again sometimes within minutes, sometimes not for hours.

Shit though, what an awful attitude by Jazzzed. I'm a dev in the Sonic Community of all places, I'm no stranger to this kind of treatment so where can I donate to make up for him _not_ donating? ^.^

**Comment by pyrates999** [comment]
Ok, there's some confusion here.  I asked you to uninstall EP and meant for you to not reinstall it.  Then check if this is still happening.

**Comment by PimpUigi** [comment]
Oh!!! Will do,

**Comment by PimpUigi** [comment]
Okay after doing that, what seems to be happening is the widgets button will display the weather for some time, and then it changes to currency or traffic (neither of which I'm interested in so that's kind of dumb on their part); do you think it's possible that when Windows calls that function that it's changing the Explorer Patcher taskbar's widget into the People button?

**Comment by pyrates999** [comment]
Might be.  Thank you for doing this.

**Comment by NifTTY** [comment]
Able to replicate this bug, with the widget keeping the size/shape of the weather. 
In our case it showed current weather until the cursor hovered over the space, after which it turned into this odd People widget.

Restarting explorer at any point (using the shortcut in EP properties window) is an easy way to replicate the bug.

![image](https://github.com/valinet/ExplorerPatcher/assets/144926478/e2d64ca1-a01c-44dc-a075-b8a9738b9d2d)
(We are also using Classic Shell / Classic Start Menu which is responsible for skinning the Taskbar with XP Luna theme)

This frequently manifested with an issue we've encounred with explorer restarting when launched after all file browser windows are closed.
After the unexpected restart, we see the same People button (fully functional People widget) showing up in place of the Weather widget. 

Check out our issue posted here:

https://github.com/valinet/ExplorerPatcher/issues/2055


Unclear if the two are related.

**Comment by pyrates999** [comment]
> Able to replicate this bug, with the widget keeping the size/shape of the weather. In our case it showed current weather until the cursor hovered over the space, after which it turned into this odd People widget.
> 
> Restarting explorer at any point (using the shortcut in EP properties window) is an easy way to replicate the bug.
> 
> ![image](https://user-images.githubusercontent.com/144926478/267719170-e2d64ca1-a01c-44dc-a075-b8a9738b9d2d.png) (We are also using Classic Shell / Classic Start Menu which is responsible for skinning the Taskbar with XP Luna theme)
> 
> This frequently manifested with an issue we've encountered with explorer restarting when launched after all file browser windows are closed. After the unexpected restart, we see the same People button (fully functional People widget) showing up in place of the Weather widget.
> 
> Check out our issue posted here:
> 
> #2055
> 
> Unclear if the two are related.

Classic Shell is no longer maintained.  Install openshell instead.

**Comment by NifTTY** [comment]
> Classic Shell is no longer maintained. Install openshell instead.

Pardon the confusion. Previous comment edited.

**Comment by pyrates999** [comment]
What setting in Open Shell are you using to apply the luna theme?

**Comment by NifTTY** [comment]
> What setting in Open Shell are you using to apply the luna theme to the taskbar?

Toggle "Show All Settings" then go to the "Taskbar" section.
Click the "..." next to "Taskbar texture"
You will be prompted to select a file.

![luna_taskbar](https://github.com/valinet/ExplorerPatcher/assets/144926478/c73580d9-3ce6-48c0-8686-d9dbd820fdce)


Works great with EP on Win11. A nice combination.
Does not work consistently in vanilla Win10 after 20H2.

**Comment by pyrates999** [comment]
Interesting.  I didn't know about this feature in open shell.

**Comment by NifTTY** [comment]
In our experience, this bug usually manifests when restarting explorer.
Rarely does it spontaneously change before EP restart.

**Comment by grimmute** [comment]
I can confirm that this also happens on my side on latest stable EP on Windows 11 23H2, after I clicked "Restart File Explorer*" on settings after applying my settings.

Restarting it once again, now it's back to Weather.. but for how long I will see.

**Comment by philip47** [comment]
Still waiting for a fix..

**Comment by ENTPRESTIGIOUS** [comment]
I have the same issue.

**Comment by evieamity** [comment]
[ExplorerPatcher_22631.5335.68.2.reg.zip](https://github.com/user-attachments/files/21570136/ExplorerPatcher_22631.5335.68.2.reg.zip)

Hiya, I apologize that I don't have a solution to give and just have my version numbers and info, but as it seems a resolution has not yet been found to this issue yet, I wanted to post what I could since more data could possibly help.

I'm basically having that issue where if you run your mouse cursor over the weather widget, it sometimes turns into the people icon. Of course, restarting explorer helps, but it only happens again a random amount of time later.

Explorer Patcher: 22631.5335.68.2
Windows 11 Pro: 24H2 Build 26120.4520 (Release Preview)
Open-Shell Menu: 4.4.196 (In case that's relevant in some way)

I saw in a previous comment that you told someone to swap to the full release and not the release preview, however:

- They confirmed the issue happened in both versions. (plus I'm seeing reports of people getting this issue on the official release anyway)
- I am unable to swap to the full release as I am in the unique position of dealing with the dreaded error 0x8007042b because of "not supported hardware," while being on a release preview. I run Windows 11 anyway because my CPU _does_ meet the requirements, except MS said eff you to manually approving the i7-7700k despite it meeting requirements due to its age being one year before the cutoff. Turns out, if you are using the update bypass, you are unable to use said bypass to return to the full release from preview... Basically I'm stranded on the preview and stuck being on the Windows Insider (chump) Program (still getting updates to the OS though) >~<
- Long story short, I'm quite certain my OS isn't the cause. Thank you for listening to the rant :3

Since I needed to use the solution posted [here](https://github.com/valinet/ExplorerPatcher/issues/4351#issuecomment-3042345865) as my people icon was previously a news icon without internet, I have already tried the process of clearing the weather widget local data.

Also, a long time ago, I used a [regedit tweak](https://www.tenforums.com/tutorials/104877-enable-disable-people-bar-taskbar-windows-10-a.html) to remove the people icon, back when it was annoying me in Windows 10. I had to undo that tweak recently to get the people icon to show for this weather thingy to display. Not sure if it's relevant, but I wanted to post this info anyway.

I appreciate any help you are able to offer. I'm posting a pic of the issue to clarify what exactly is happening. I just want to say thank you to all of the super helpful devs here who have been extremely kind and patient in these threads. I notice it, and I really appreciate it. Y'all are amazing!~ ♥

<img width="637" height="677" alt="Image" src="https://github.com/user-attachments/assets/908102ff-f40f-4926-b966-c9d275559df3" />

**Comment by martinfabian** [comment]
I have this issue now on a new laptop running W11 25H2 (26200.7840), with the Windows 10 (ExplorerPatcher) taskbar style. As described above, weather is shown for some time, but then at some point when I open the laptop lid the People icon is shown and the only way to get weather back (it seems) is to restart File Explorer through the link at the bottom left in the EP Properties dialog.

On my desktop, running W11 25H2 (26200.8037), this issue not present.
